addons/point_of_sale/models/pos_order.py

```diff
@@ -753,6 +763,9 @@ def _order_line_fields(self, line, session_id=None):
         readonly=True, required=True)
     price_subtotal_incl = fields.Float(string='Subtotal', digits=0,
         readonly=True, required=True)
```

Comment:
margin and cost should be fields.Monetary
margin_percent should be fields.Float(digits="Discount")

Reply:
done